docs/notebooks/entropic_map_estimator.ipynb

```diff
@@ -0,0 +1,417 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.            ###we assume the points are sampled uniformly
</pre><p>remove this </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>